### PR TITLE
[[ Merge 6.7.7 rc 2 ]] Merge release-6.7.7 into develop-7.0

### PR DESCRIPTION
**Pull request by livecodesebastien**
Nothing had to be refactored.

One conflict (`version` apart) was in `qt-mac-recorder.mm`, as a new line was added in `develop-6.7` at line 276.

### CODE REVIEW

File: engine/src/mac-qt-recorder.mm [hunk 1]
```diff
@@ -723,8 +723,17 @@ static bool available_compression_ids(void *context, unsigned int p_id, const ch
...
     exportToSoundFile(m_temp_file, m_filename);
-    
+	MCS_unlink(m_temp_file);
+
+	if (m_filename != nil)
+		MCCStringFree(m_filename);
+
+	if (m_temp_file != nil)
+		MCCStringFree(m_temp_file);
+
```
Comment (livecodeali) [inline engine/src/mac-qt-recorder.mm:755]: These should be MCValueAssigned the empty string I think

**Comment by livecodeali** [comment]
@livecode-vulcan review ok  821d1f0

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 821d1f09c90bca9e616f222334e8a7d0b836715a

**Comment by livecode-vulcan** [comment]
😎 test success 821d1f09c90bca9e616f222334e8a7d0b836715a
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/685)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/744)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/634)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/698)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/630)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/622)

**Pull request merged**